### ISSUE TITLE
128870069 Coordinator response emails (Team)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1171]
Issue reported by **alycymru** at 2019-01-31 13:14:29 UTC
Time: 2019-01-31T13:08:29+00:00
Session ID: 52dc75c1ffc867067ff998748b318221
Problem Page URL: [/messages/userid_messages](/messages/userid_messages)
Previous Page URL: [https://www.freereg.org.uk/manage_resources/new](https://www.freereg.org.uk/manage_resources/new)
Reported Issue:
I have just noticed that when I reply to a contact, my reply shows up as a message on my home screen. It also has provide my 'private' email to those contacts, as I just had a lady reply to me, but it did not show up on FR, but rather in my gmail account. Thought this was not supposed to be happening
https://www.freereg.org.uk/uploads/feedback/screenshot/5c52f4b5f493fdf08da27c11/FR.jpg    

![Screenshot](https://www.freereg.org.uk/uploads/feedback/screenshot/5c52f4b5f493fdf08da27c11/FR.jpg)

**Comment by Sherlock21** [comment]
Forwarded to Kirk

reason: I clicked on SHOW, to see what this strange message was about, and a sytem message just says: NOT FOUND.

?? something gone off track?

Eric b
On 1 Feb 2019, at 16:29, FreeUKGenIssues <notifications@github.com> wrote:

Issue reported by alycymru at 2019-01-31 13:14:29 UTC
Time: 2019-01-31T13:08:29+00:00
Session ID: 52dc75c1ffc867067ff998748b318221
Problem Page URL: /messages/userid_messages
Previous Page URL: https://www.freereg.org.uk/manage_resources/new
Reported Issue:
I have just noticed that when I reply to a contact, my reply shows up as a message on my home screen. It also has provide my 'private' email to those contacts, as I just had a lady reply to me, but it did not show up on FR, but rather in my gmail account. Thought this was not supposed to be happening
https://www.freereg.org.uk/uploads/feedback/screenshot/5c52f4b5f493fdf08da27c11/FR.jpg

**Comment by suffolkroots** [comment]
![email address](https://user-images.githubusercontent.com/10379655/52163507-f19f8480-26da-11e9-8dc3-b0c7af2dd776.jpg)
alycymru is correct---our personal email address is on replies. There is no indication on FR that this is the case. It only becomes obvious when a reply is sent to our personal email address.

**Comment by Captainkirkdawson** [comment]
This was the design criteria improve delivering of messages and to enhance personal communications. It uses the members email address.

**Comment by suffolkroots** [comment]
This seems to nullify your intention to have all communications recorded on FR, as, if the contact subsequently replies to my address, we can complete the communication without it ever reaching FR.

And, if this is how you've set it up, you might as well forward the initial contact to my address.

**Comment by alycymru** [comment]
Just to be clear. 

#1 The copy of the reply I sent to the contact which I submitted as a screenshot appeared as a message on when I opened FR., the same as when messages are sent via the system to inform us of changes, etc. That did not start happening until we had the last update I believe. Have not had a new contact since the reset to test this.

#2 As stated by 'suffolkroots' above, If a contact responds to our reply, it is being sent straight to our private emails and does not go through the system on FR, so this further contact is not being recorded. Again, I was under the impression that everything was being recorded in FR, and we were not needing to make use of our private email address in any respect. I do not remember us being told that the new system was making use of our private email address for anything other than to notify us that we had a contact.,, or I would have voiced a concern at that time. Agree it completely negates the idea of what FR was trying to do in the first instance.

**Comment by Sherlock21** [comment]
I agree with the sentiment expressed above. there should be no personal email addresses coming into play
although I had not particularly noticed myself.

**Comment by Sherlock21** [comment]
I agree with the sentiment expressed above. there should be no personal email addresses coming into play
although I had not particularly noticed myself.

**Comment by PatReynolds** [comment]
The deliverability of emails is very important. But so is privacy. I believe both are not incomparable.

**Comment by SteveBiggs** [comment]
The main motivation for having coordinators respond to researchers' queries through the FR website is so we can see that the issue has been addressed. In most cases, the initial response from the coordinator will resolve the issue. However, if the researcher wants to make a supplementary question (or just to say thank you), they cannot do this through the website message system (as they don't have a userid) so the coordinator's email address is provided to allow this.

If coordinators don't want their personal email address made available to researchers, we have offered the use of a freeukgenealogy hosted address such as ssx.freereg@freeukgenealogy.org.uk. As well as separating FR emails from your personal communications, this has the benefit that when a coordinator steps down, they can hand over the email address to the new coordinator and it will have all the history of their FR communications.
